src/libusb/enumerator-libusb.cpp

```diff
@@ -80,7 +80,7 @@ namespace librealsense
         std::vector<usb_device_info> usb_enumerator::query_devices_info()
         {
             std::vector<usb_device_info> rv;
-            auto ctx = std::make_shared<usb_context>();
+            auto ctx = get_usb_context();
```

Comment:
I went to clean up the fact that we are creating a usb_context every time we list devices, but before I could fix it, I noticed that @bfulkers-i had already fixed this as part of #5492!

Reply:
I plan to rebase #5492 on top of this once this gets merged.